src/components/units-list/unit-chart/pie-chart/pie-chart.utils.ts

```diff
@@ -17,27 +19,19 @@ export const getShortNumber = (value: number): string => {
   return `${shortedValue} mln`
 }
 
-export const colors: string[] = [
-  theme('colors.blue.bright'),
-  theme('colors.purple.bright'),
-  theme('colors.yellow.secondary'),
-  theme('colors.orange.bright'),
-  theme('colors.grey.blue'),
-]
-
-const maxItems = colors.length
-
-export const getPieChartItemColor = (index: number): string =>
-  colors[index] ?? colors[colors.length - 1]
+const maxItems = defaultColors.length
 
 export const drawPieChart = ({
   svg,
   data,
   width,
   height,
   tooltip,
+  isDashboard,
+  selectedVariants,
+  onSelectVariantByChart,
 }: SvgChartRenderParams<TPieChartData>): void => {
-  const radius = Math.min(width, height) / 2
+  const radius = Math.min(width, height) / 2.02
```

Comment:
Why 2.02?

Reply:
<img width="361" alt="Screenshot 2022-08-22 at 15 54 04" src="https://user-images.githubusercontent.com/74919871/185915365-adf1f673-0ef1-4227-bcf1-0f6204726b7e.png">
To get rid of this cutting